### PR TITLE
Add documentation note on RTD-Binder incompatibility

### LINKED ISSUE #503: Binder integration and Read the docs
Hi, after doing some tinkering trying to incorporate binder links into our into our example gallery - am I right in saying that there's currently no way of getting that working with readthedocs as a documentation builder, since binder needs a repository with hosted notebooks? If that's the case, I'll update the docs with a note on that.

### PR DESCRIPTION
**Pull request by StanczakDominik**
Closes #503

### CODE REVIEW

**Comment by StanczakDominik** [comment]
Oh dear, I think @codecov-io may have overworked itself.

File: doc/configuration.rst [hunk 1]
```diff
@@ -660,6 +660,8 @@ specified in ``notebooks_dir``. This will be a subfolder of the sphinx output
 directory and included with your site build.
 Binder links will point to these notebooks.
 
+Please note that it is currently impossible to host notebooks generated by Sphinx-Gallery via readthedocs.org, as it does not provide you with a GitHub repository you could link Binder to.
```
Comment (choldgraf) [inline doc/configuration.rst]:
```suggestion
.. note::

   It is not currently possible to host notebooks generated by
   Sphinx-Gallery with readthedocs.org, as RTD does not provide you
   with a GitHub repository you could link Binder to. If you'd like to
   use readthedocs with Sphinx-Gallery and Binder links, you should
   independently build your documentation and host it on a GitHub branch
   as well as building it with readthedocs.
```

Comment (choldgraf) [reply doc/configuration.rst]: nice - thanks for the note! here's a quick suggestion

**Comment by choldgraf** [comment]
LGTM! Thanks for the docs patch :-)  and I have no idea what happened to codecov - maybe it just got overzealous 😆

**Pull request merged**